modules/social_features/social_content_block/social_content_block.module

```diff
@@ -56,25 +56,50 @@ function _social_content_block_is_custom_content_block(BlockPluginInterface $blo
 }
 
 /**
- * Implements hook_form_BASE_FORM_ID_alter().
+ * Implements hook_form_alter().
  */
-function social_content_block_form_block_content_custom_content_list_form_alter(&$form, FormStateInterface $form_state, $form_id) {
-  // Add general description above first field to explain what the filters do.
-  $form['field_topic_type'] += [
-    '#prefix' => t('To make the list of topics more specific you can additionally configure more filters such as topic types, content tags and groups.'),
```

Comment:
This removed translation should be added to the translations.php file with a proper description.

Additionally, this description is not brought back so should we add something somewhere to show the user what should be done?

Reply:
According to the new design it doesn't need anymore. So I added these phrases to the translations.php file